test/kafka_connect_data_onboarding_tests/test_data_onboarding.py

```diff
@@ -1,9 +1,41 @@
 import logging
 import sys
+import pytest
+from ..commonsplunk import check_events_from_splunk
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 formatter = logging.Formatter('%(message)s')
 handler = logging.StreamHandler(sys.stdout)
 handler.setFormatter(formatter)
-logger.addHandler(handler)
\ No newline at end of file
+logger.addHandler(handler)
+from ..commonkafka import *
+
+
+class TestDataOnboarding:
+
+    @pytest.fixture(scope='class', autouse=True)
+    def setup_class(self, setup):
+        pytest_configure(setup)
```

Comment:
why have you moved pytest_configure function to commonkafka and call it in each class? then the connectors has to be deployed and deleted and redeployed again, right? every deployment, we added a wait time of 80 sec. now we have to wait for 160 sec, right?

Reply:
yes, I can move it back.